velox/exec/fuzzer/WindowFuzzer.cpp

```diff
@@ -406,6 +610,8 @@ bool WindowFuzzer::verifyWindow(
 
     if (!customVerification) {
       if (resultOrError.result && enableWindowVerification) {
+        referenceQueryRunner_->queryRunnerContext_->windowFrames_[windowNodeId]
+            .push_back(prestoFrameClause);
```

Comment:
Do we clear referenceQueryRunner_->queryRunnerContext_->windowFrames_ after every fuzzer iteration?